gcmr/assembly/transient/Griffin_transient.i

```diff
@@ -0,0 +1,220 @@
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = '../MESH/Griffin_mesh.e'
```

Comment:
Most of the comments on the steady state input file apply here.

Reply:
Thank you, applied them to the transient input file as well.